### PR TITLE
[LD-815] Addition of mixed_companies search

### PR DESCRIPTION
**Pull request by Kavya-24**
Jira: https://apollopde.atlassian.net/browse/LD-815

Make sure you've checked off all these things before submitting:

- [x] This pull request isn't for a company's fork, it's intended for the upstream Slate shared by everybody.
- [x] This pull request is submitted to the `master` branch.
- [x] If it makes frontend changes, this pull request has been tested in the latest version of Firefox, Chrome, IE, and Safari.
- [x] Tested that shell and python code are working as expected

Screenshot 2023-10-19 at 1 54 23 PM
Screenshot 2023-10-19 at 1 54 33 PM
Screenshot 2023-10-19 at 1 54 39 PM
Screenshot 2023-10-19 at 1 54 49 PM

### CODE REVIEW

**Comment by PavanSankavar** [comment]
Label : " # employees" is repeating in response

File: source/includes/_organizations.md [hunk 1]
```diff
@@ -77,3 +77,187 @@ Get a list of active job postings for a company.
...
+            "value": "Apollo.io",
+            "display_name": "Apollo.io"
+        },
+        {
+            "label": "# Employees",
+            "signal_field_name": "organization_num_employees_ranges",
+            "value": "1,100",
+            "display_name": "1-100"
+        },
+        {
```
Comment (PavanSankavar) [inline source/includes/_organizations.md:141]: This is repeating, value has been changed. is this intentional ?

File: source/includes/_organizations.md [hunk 2]
```diff
@@ -77,3 +77,187 @@ Get a list of active job postings for a company.
...
+            "organization_state": "California",
+            "organization_country": "United States",
+            "organization_postal_code": "94105",
+            "suggest_location_enrichment": false,
+            "parent_account": {
+                "id": "62b5283531e8210001210XXX",
+                "name": "Apollo.io"
+            },
+            "domain": "apollo.io",
+            ...
```
Comment (PavanSankavar) [inline source/includes/_organizations.md:213]: Is this expected ?

**Review by PavanSankavar — COMMENTED** [review]
Added few review comments, please check

File: source/includes/_organizations.md (hunk 1 above)
Comment (Kavya-24) [reply source/includes/_organizations.md:141]: These are auto-constructed from the breadcrumbs. Does that answer?

File: source/includes/_organizations.md (hunk 2 above)
Comment (PavanSankavar) [reply source/includes/_organizations.md:213]: we should get this confirmed from product.

File: source/includes/_organizations.md (hunk 1 above)
Comment (PavanSankavar) [reply source/includes/_organizations.md:141]: Okay, just want to understand, whether this has been added by mistake

File: source/includes/_organizations.md (hunk 2 above)
Comment (atul-tw) [reply source/includes/_organizations.md:213]: I think documentation should have all the fields as it helps someone to integrate that in their systems

**Review by atul-tw — APPROVED** [review]
I am good with the PR apart from the trimming of the response. Please check with product on that...!!!

File: source/includes/_organizations.md (hunk 1 above)
Comment (Kavya-24) [reply source/includes/_organizations.md:141]: Can we resolve this @PavanSankavar

File: source/includes/_organizations.md (hunk 2 above)
Comment (Kavya-24) [reply source/includes/_organizations.md]: Removed the` ...` and added all the fields.

Comment (Kavya-24) [reply source/includes/_organizations.md]: Checked across other places in documentation, we are not essentially removing the fields, just reducing the values if the response is too large. Made consistent with that behaviour.

**Review by PavanSankavar — APPROVED** [review]
LGTM

**Comment by Kavya-24** [comment]
Screenshot 2023-10-23 at 5 25 17 PM

**Pull request merged**